apps/foundation-demo/src/app/cart/components/cart-summary/cart-summary.component.spec.ts

```diff
@@ -58,52 +58,52 @@ describe('CheckoutCartComponent', () => {
     });
 
     wrapper.cartValue = mockCart;
-    wrapper.subtitleValue = stubSubtitle;
+    wrapper.titleValue = stubTitle;
 
     fixture.detectChanges();
 
-    checkoutCartItems = fixture.debugElement.queryAll(By.css('demo-checkout-cart-item'));
-    checkoutCart = fixture.debugElement.query(By.css('demo-checkout-cart')).componentInstance;
+    cartSummaryItems = fixture.debugElement.queryAll(By.css('demo-minicart-item'));
```

Comment:
I think this should be minicartItems, because that's how it's referenced in the html

Reply:
Resolved in 08c6569